docs/samples/host-a-webgame.md

```diff
@@ -1,29 +1,39 @@
 # Hosting a web game on the Internet Computer
 
-This sample project demonstrates how to deploy a web game on the [Internet Computer](https://github.com/dfinity/ic) through [Unity](https://unity.com/) and [Godot](https://godotengine.org/) game engines. The sample `dfx` projects are available at the [examples](https://github.com/dfinity/examples/tree/master/hosting) repository.
+This sample project demonstrates how to deploy a web game on the [Internet Computer](https://internetcomputer.org/) through [Unity](https://unity.com/) and [Godot](https://godotengine.org/) game engines.
 
-Before you begin, make sure you have [dfx SDK](https://smartcontracts.org/docs/quickstart/local-quickstart.html#download-and-install) installed.
+Before you begin, make sure you have [dfx SDK](https://internetcomputer.org/docs/current/developer-docs/build/install-upgrade-remove) installed.
```

Comment:
```suggestion
Before you begin, make sure you have [dfx SDK](/docs/current/developer-docs/build/install-upgrade-remove.md) installed.
```
If it's an internal link CI will notice if it breaks.

Adding file extension because otherwise we have problems with 404s